content/hardware/04.pro/carriers/portenta-max-carrier/tutorials/mpcie-4g-modem/content.md

```diff
@@ -312,16 +314,74 @@ fw_setenv is_on_carrier yes
 fw_setenv carrier_name max
 ```
 
-The previous commands are used to set environment variables, which we will use to set and use the needed overlays to link the mini PCIe interface under its profile. The overlays are as follows and set using the following command:
+The previous commands are used to set environment variables, which we will use to set and apply the needed overlays to link the USB modem interface under its profile. The overlays are as follows and set using the following command:
 
 ```bash
-fw_setenv overlays 'ov_som_lbee5kl1dx ov_som_x8h7 ov_carrier_enuc_bq24195 ov_carrier_max_usbfs ov_carrier_max_sdc ov_carrier_max_cs42l52 ov_carrier_max_pcie_mini'
+fw_setenv overlays 'ov_som_lbee5kl1dx ov_som_x8h7 ov_carrier_enuc_bq24195 ov_carrier_max_usbfs ov_carrier_max_sdc ov_carrier_max_cs42l52'
 ```
 
+***The `ov_carrier_max_pcie_mini` overlay is not required for USB modems such as the GNSS Global (EG25) and EMEA (EC200A-EU) variants of the Pro 4G Module.***
+
 Once the overlays are set, please reboot the Portenta X8 to ensure the configuration has been applied correctly.
 
 ### Connecting & Testing Network Connectivity
 
+#### GNSS Global EG25 Module
+
+For the **GNSS Global (EG25) Module**, you can configure the modem using **nmcli**:
```

Comment:
Clarify that nmcli is NetworkManager and that this integration is possible since this modem supports QMI interface